src/Captcha.php

```diff
@@ -27,6 +27,12 @@ abstract public function renderHeadTag();
 
     public function verify()
     {
+        if (request()->boolean('disable_captcha', false)) {
```

Comment:
Let's add `request()->ajax()` to this check to ensure injecting an input in the form before submission doesn't skip captcha verification.

Reply:
Good call